maintainers/maintainer-list.nix

```diff
@@ -5516,6 +5516,11 @@
     github = "vskilet";
     name = "Victor SENE";
   };
+  vyorkin = {
```

Comment:
Please separate this PR into 2 commits. The first commit should be adding yourself to maintainers list file, and the second being the derivation.